addons/mail/static/src/core/web/out_of_focus_service_patch.js

```diff
@@ -3,21 +3,42 @@ import { patch } from "@web/core/utils/patch";
 import { _t } from "@web/core/l10n/translation";
 import { sprintf } from "@web/core/utils/strings";
 
+export const UNREAD_MSG_TITLE = 100;
+
 patch(OutOfFocusService.prototype, {
     setup(env, services) {
         super.setup(env, services);
         this.titleService = services.title;
         this.counter = 0;
-        env.bus.addEventListener("window_focus", () => {
-            this.counter = 0;
-            this.titleService.setParts({ _chat: undefined });
+        env.bus.addEventListener("window_focus", () => this.clearUnreadMessage());
+        this.intervalShowing = true;
+    },
+    clearUnreadMessage() {
+        this.counter = 0;
+        this.titleService.setParts({ [UNREAD_MSG_TITLE]: undefined });
+        this.intervalShowing = true;
+        this.titlePattern = undefined;
+        clearInterval(this.newMessageInterval);
+    },
+    setInterval(func, duration) {
+        return setInterval(func, duration);
+    },
+    updateTitle() {
+        this.titleService.setParts({
+            [UNREAD_MSG_TITLE]: this.intervalShowing
+                ? sprintf(this.titlePattern, this.counter)
+                : undefined,
         });
+        this.intervalShowing = !this.intervalShowing;
     },
     notify() {
         super.notify(...arguments);
         this.counter++;
-        const titlePattern = this.counter === 1 ? _t("%s Message") : _t("%s Messages");
-        this.titleService.setParts({ _chat: sprintf(titlePattern, this.counter) });
+        this.titlePattern = this.counter === 1 ? _t("%s Message") : _t("%s Messages");
+        this.updateTitle();
+        this.newMessageInterval = this.setInterval(() => {
+            this.updateTitle();
+        }, 1_000);
     },
```

Comment:
@alexkuhn Does calling `notify()` multiple times instantiate multiple intervals ? Is there a case where a new interval is assigned to `this.newMessageInterval` and you can lose the possibility to clear it in `clearUnreadMessage()` ?